tutorial/chapter3.jade

```diff
@@ -47,9 +47,9 @@
 
 .step(data-title='Notifying the Guest', data-file="app/models/reservation.rb", data-highlight="43-50")
   :markdown
-    The final step in our workflow is to notify the guest that their reservation has been booked. Well remember the second reservation callback from a previous step? Now we get to use it! 
+    The final step in our workflow is to notify the guest that their reservation has been booked. Well remember the second reservation callback from a previous step? Now we get to use it!
 
-    By using the `after_update` callback we gaurantee that this method is going to run. It's quite simple:
+    By using the `after_update` callback we guarantee that this method is going to run. It's quite simple:
```

Comment:
Do you know how hard I had to try to avoid changing "center" to "centre" when going through this?!